rampwf/workflows/SKLearnPipeline.py

```diff
@@ -0,0 +1,27 @@
+from ..utils.importing import import_file
+import pandas as pd
```

Comment:
import pandas first

also file names should be snake_case so I would call this file sklearn_pipeline.py

Reply:
Yes, definitely sklearn_pipeline.